testinfra/docker-compose-test.yml

```diff
@@ -14,7 +14,7 @@ services:
     command:
       - /var/besu
   besu:
-    image: hyperledger/besu:latest
+    image: hyperledger/besu:24.10.0
```

Comment:
I found `latest` on my laptop was an old release, that didn't have the empty blocks modification. So it seemed good for us all to be testing with a consistent release.